components/modals/oddAttachModal.tsx

```diff
@@ -6,7 +6,11 @@ import { opModal } from "../serviceMap/opModalSignal.ts";
 import { opUpdateSignal } from "../../islands/serviceMap.tsx";
 
 export const OddAttachModal = (
-  { serviceMap }: { serviceMap: ServiceMap },
+  { serviceMap, setAttachSelectOpen, attachSelectOpen }: {
+    serviceMap: ServiceMap;
+    setAttachSelectOpen: () => void;
```

Comment:
I'm a bit confused by the multiple verbs here `attach` and `select`. It this is just tracking if the the attach modal is open then maybe just remove `select`.